common/focuses/wc_magic_lifestyle_focuses.txt

```diff
@@ -0,0 +1,641 @@
+﻿light_magic_focus_1 = {
```

Comment:
I'll leave it here as a reminder.

I guess we have to lock magics with `can_start_new_lifestyle_tree_trigger` like other perks. I'm checking characters. If they have many magic lifestyles open, they prefer to spend perks on them, not on their education tree.

That's Magni for example.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/119024327-0095c800-b9b4-11eb-8208-c22e284f71f0.png)
![image](https://user-images.githubusercontent.com/46384992/119024360-08ee0300-b9b4-11eb-988c-58afecdc430d.png)
![image](https://user-images.githubusercontent.com/46384992/119024375-0c818a00-b9b4-11eb-907c-117a69b085f9.png)

</details>

I have another idea why it doesn't work.

This is a diplomacy guy.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/119024494-2cb14900-b9b4-11eb-9024-ca90568ce0e9.png)

</details>

So I guess he took diplomacy perk as it should but then with very small random picked a starting light perk. And then... auto-selection is the default, so he continued picking light perks since they don't have auto-selection scripted.

So I guess we either have to use `can_start_new_lifestyle_tree_trigger` or try to add auto-selection weight to every perk.

Reply:
> or try to add auto-selection weight to every perk

Doesn't work

![image](https://user-images.githubusercontent.com/46576860/120054346-1a7c7e00-c038-11eb-89c8-44a3b4f448c3.png)

Reply:
> I guess we have to lock magics with can_start_new_lifestyle_tree_trigger like other perks.

Added, now I need to fix focuses